src/pybind/mgr/dashboard/controllers/hardware.py

```diff
@@ -0,0 +1,21 @@
+
+from typing import List, Optional
+
+from ..services.hardware import HardwareService
+from . import APIDoc, APIRouter, Endpoint, EndpointDoc, RESTController
+
+
+# pylint: disable=R0904
+@APIRouter('/hardware')
+@APIDoc("Hardware management API", "Hardware")
+class Hardware(RESTController):
+
+    @Endpoint()
+    @EndpointDoc("Retrieve a summary of the hardware health status")
+    def health(self, categories: Optional[List[str]] = None, hostname: Optional[List[str]] = None):
```

Comment:
let's call it summary since that's what this endpoint actually provides. and also lets' version these endpoints to EXPERIMENTAL to make sure that this is not fully ready to be consumed.

```suggestion
    def summary(self, categories: Optional[List[str]] = None, hostname: Optional[List[str]] = None):
```

Reply:
I think that what we are retriving is a health summary, that's why I went with health, in the end we are checking the health of each hardware item